custom/ilsgateway/templates/ilsgateway/multi_report.html

```diff
@@ -72,10 +72,6 @@ <h4 class="media-heading">{{ thisreport.report_table.title }}</h4>
 {% block js-inline %}
     <script>
         $(document).ajaxStop(function () {
-            $('#group_location_async select').first().css('display', 'none');
-            if ($('#group_location_async select').length === 5) {
-                $('#group_location_async select').last().css('display', 'none');
-            }
```

Comment:
We don't want to have facilities in location filter so instead of hiding them in js I exclude facilities on the backend.